src/v/reflection/type_traits.h

```diff
@@ -18,6 +18,7 @@
 #include <seastar/core/circular_buffer.hh>
```

Comment:
>  serde/serde.h: added support for std::array as size,elements (+ throw)


what does `+ throw` mean?

Reply:
It means that it will throw a serde_exception if the serialized size does not match the std::array size